code/__DEFINES/preferences.dm

```diff
@@ -29,6 +29,8 @@
 #define UI_DARKMODE 131072
 #define DISABLE_KARMA 262144
 
+#define TYPING_ONCE 1048576
```

Comment:
Done to be compatible with my other PR

Reply:
https://github.com/ParadiseSS13/Paradise/blob/e57f0b9306060be7ecd6177db7e5a674adbc96ce/code/modules/client/preference/preferences_mysql.dm#L58 You need to increase this
Replace `524287` with `2097151`